spring-cloud-dataflow-server-cloudfoundry-docs/pom.xml

```diff
@@ -73,6 +73,41 @@
 						<groupId>org.asciidoctor</groupId>
 						<artifactId>asciidoctor-maven-plugin</artifactId>
 						<version>1.5.3</version>
+						<dependencies>
+							<dependency>
+								<groupId>org.asciidoctor</groupId>
+								<artifactId>asciidoctorj-pdf</artifactId>
+								<version>1.5.0-alpha.11</version>
+							</dependency>
+						</dependencies>
+						<configuration>
+							<sourceDocumentName>index.adoc</sourceDocumentName>
+							<doctype>book</doctype>
+							<attributes>
+								<allow-uri-read>true</allow-uri-read>
+								<attribute-missing>warn</attribute-missing>
+								<project-version>${project.version}</project-version>
+								<project-artifactId>${project.artifactId}</project-artifactId>
+								<version-type-lowercase>${version-type-lowercase}</version-type-lowercase>
+
+								<dataflow-project-version>${spring-cloud-dataflow.version}</dataflow-project-version>
+								<deployer-cloudfoundry-version>${spring-cloud-deployer-cloudfoundry.version}</deployer-cloudfoundry-version>
+								<dataflow-version-type-lowercase>${dataflow-version-type-lowercase}</dataflow-version-type-lowercase>
+								<dataflow-branch-or-tag>${dataflow-branch-or-tag}</dataflow-branch-or-tag>
+								<dataflow-asciidoc>https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/${dataflow-branch-or-tag}/spring-cloud-dataflow-docs/src/main/asciidoc</dataflow-asciidoc>
+								<deployer-branch-or-tag>${deployer-branch-or-tag}</deployer-branch-or-tag>
+
+								<!-- The following two are deps of this project and set automatically.-->
+								<!-- Others are loose dependencies, and current-SNAPSHOT typically make sense-->
+								<scdf-core-version>${org.springframework.cloud:spring-cloud-dataflow-core:jar.version}</scdf-core-version>
+								<sct-core-version>${org.springframework.cloud:spring-cloud-task-core:jar.version}</sct-core-version>
+								<scdt-core-version>current-SNAPSHOT</scdt-core-version>
```

Comment:
As the comment above (I initially wrote) states, this is fine. This is just used to explain concepts of SCST.